### ISSUE TITLE
Heya!

### ISSUE THREAD

**Issue by GeorgeBgk** [RealPipeline/FileShare#1]
Heya! I found you on Scratch.
Let's get to the point. You have inline CSS styling. 
`<p style="color:white;"></p>`
That is kinda wrong.
Try styling the other way

In head tag:
`<style>`
`#parag {`
`   color: white;`
`}`
`</style>`

In body tag:
`<p id="parag"></p>`

**Comment by ALinuxLover** [comment]
True, or just have an external CSS file. Either way is better than online styling.
